src/main/java/com/khutircraftubackend/controllers/SellerController.java

```diff
@@ -0,0 +1,9 @@
+package com.khutircraftubackend.controllers;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/v1/api")
```

Comment:
normally, we don't put "/api" in the backend itself. 
`/api` serves to be a proxy path from front end to backend, and is configured at the frontend.
here, at this place, it's better to have `/v1/seller`